### DISCUSSION TITLE
iOS app missing much needed functionality

### DISCUSSION THREAD

**Discussion by haikusw** [community/community#48238] [Enterprise] [answered]
- Ability to easily set it up to use two accounts neither of which are enterprise accounts.
- Ability to switch between these two accounts easily.
- Ability to fork a repo (huh?)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.99.0 (90728769)
Device: iPhone 12 Pro (iOS 15.7.1)
</details>

**Comment by candyho** [comment, marked as answer]
Hey, thanks for sharing the feedback! Multi-account support is something that we've planned to work on. Stay tuned for more update.

I have added your request to fork a repo to our internal issue for tracking purpose. Thanks!
